### ISSUE TITLE
Update adapter script for new testing suite

### ISSUE THREAD

**Issue by jtcohen6** [dbt-labs/dbt-core#4929]
Either:
- Update the existing script to instantiate `pytest`-based adapter basic tests (added in https://github.com/dbt-labs/dbt-core/pull/4846), rather than `.dbtspec` file
- Replace the existing script with the interactive scaffold that we worked on last fall (https://github.com/dbt-labs/dbt-core/issues/4281), once and for all... and update the test scaffolding there too :)

**Comment by nathaniel-may** [comment]
@jtcohen6, which script are you referring to? We want to make sure we have the exact change before estimating.

**Comment by jtcohen6** [comment]
Good call! https://github.com/dbt-labs/dbt-core/blob/main/core/scripts/create_adapter_plugins.py

Documented here: https://docs.getdbt.com/docs/contributing/building-a-new-adapter

**Comment by jtcohen6** [comment]
Closing in favor of #4980, as we want to productionize the new cookiecutter-based adapter scaffold
